lib/managers/KalturaFetchManager.js

```diff
@@ -118,7 +118,8 @@ class KalturaFetchManager extends kaltura.KalturaManager
 			},
 			function (err)
 			{
-				This._failedToTranscodeFiller(adsReadyCacheKey, err);
+				if (err != 'File is currently transcoding on local disk')
```

Comment:
'File is currently transcoding on local disk' appears in many locations should be a constant